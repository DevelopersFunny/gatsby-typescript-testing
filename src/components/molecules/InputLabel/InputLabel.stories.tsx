import React from 'react'
import { ComponentStory, ComponentMeta } from '@storybook/react'
import { InputLabel } from '.'

export default {
  title: 'Atoms/InputLabel',
  component: InputLabel,
} as ComponentMeta<typeof InputLabel>

const Template: ComponentStory<typeof InputLabel> = args => (
  <InputLabel {...args} />
)

export const InputLabelName = Template.bind({})
InputLabelName.args = {
  name: 'inputlabel',
  id: 'inputlabel',
  type: 'text',
  placeholder: 'Input label',
}
