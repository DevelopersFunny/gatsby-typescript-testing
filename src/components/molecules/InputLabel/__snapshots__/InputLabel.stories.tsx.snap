// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stories storyshots Molecule/InputLabel Input Label Filled 1`] = `
<div
  className="relative"
>
  <input
    className="peer css-1xsr6mf"
    data-testid="inputFilled"
    id="inputFilled"
    name="inputFilled"
    placeholder="Input label"
    type="text"
  />
  <label
    className=" absolute -top-3.5 h-10 w-full pl-9 pr-3 text-sm text-gray-600  transition-all  peer-placeholder-shown:top-2 peer-placeholder-shown:text-base  peer-placeholder-shown:italic  peer-placeholder-shown:text-gray-400 peer-focus:-top-3.5  peer-focus:text-sm  peer-focus:text-gray-600 css-72egtx"
    htmlFor="inputFilled"
    style={
      Object {
        "backgroundColor": undefined,
        "color": undefined,
      }
    }
  >
    Input label
  </label>
</div>
`;

exports[`Stories storyshots Molecule/InputLabel Input Label Name 1`] = `
<div
  className="relative"
>
  <input
    className="peer css-1xsr6mf"
    data-testid="inputLabel"
    id="inputLabel"
    name="inputLabel"
    placeholder="Input label"
    type="text"
  />
  <label
    className=" absolute -top-3.5 h-10 w-full pl-9 pr-3 text-sm text-gray-600  transition-all  peer-placeholder-shown:top-2 peer-placeholder-shown:text-base  peer-placeholder-shown:italic  peer-placeholder-shown:text-gray-400 peer-focus:-top-3.5  peer-focus:text-sm  peer-focus:text-gray-600 css-72egtx"
    htmlFor="inputLabel"
    style={
      Object {
        "backgroundColor": undefined,
        "color": undefined,
      }
    }
  >
    Input label
  </label>
</div>
`;
